installer/parse-ros-package-deps.py

```diff
@@ -29,11 +33,12 @@ def main():
     deps = doc.findall('exec_depend')
     dep_set |= set([dep.text for dep in deps])
 
-    deps = doc.findall('test_depend')
-    dep_set |= set([dep.text for dep in deps])
+    if os.getenv('ROBOT_REAL', 'false') != 'true':  # Skip test and doc dependencies on actual bots
```

Comment:
But ROBOT_REAL will not be set, when running the installer for the first time.

Reply:
Ah yes, would it be wise to implement something that does work? 
A proper solution doesn't really come to mind at the moment..

Reply:
If we export  `ROBOT_REAL` in `install.bash` of the robot target, will this work then?

Reply:
No, as it is not the first target to be installed.

Reply:
I think we shouldn't install these by default, unless you provide an extra argument or config item.

Reply:
That's sounds like a better option.

Reply:
That could work.
In that case I would make the config item default if the 'user' target is installed? Not testing in developer mode might encourage bad practice?